PolyEngine/Engine/Src/Engine.hpp

```diff
@@ -223,6 +228,8 @@ namespace Poly
 
 		std::unordered_map<std::type_index, size_t> ComponentTypeMap;
 		std::unordered_map<std::type_index, size_t> WorldComponentTypeMap;
+
+		bool shouldQuit = 0; //stop the game
```

Comment:
According to our coding style members should be names using PascalCase, not cameCase. Also let's rename it to `GameQuitRequested` and initialize with `false` and not 0.